src/Tasks/Microsoft.NET.Build.Tasks/ResolveTargetingPackAssets.cs

```diff
@@ -24,51 +31,71 @@ public class ResolveTargetingPackAssets : TaskBase
+                        referencesToAdd.Add(reference);
+                    }
+
+                    var platformManifestPath = Path.Combine(targetingPackRoot, "build", "netcoreapp3.0",
+                        targetingPack.GetMetadata(MetadataKeys.PackageName) + ".PlatformManifest.txt");
+
+                    if (File.Exists(platformManifestPath))
+                    {
+                        platformManifests.Add(new TaskItem(platformManifestPath));
+                    }
+
+                    if (targetingPack.ItemSpec.Equals("Microsoft.NETCore.App", StringComparison.OrdinalIgnoreCase))
+                    {
+                        //  Hardcode this for now.  Load this from the targeting pack once we have "real" targeting packs
+                        //  https://github.com/dotnet/cli/issues/10581
+                        PackageConflictPreferredPackages = "Microsoft.NETCore.App;runtime.linux-x64.Microsoft.NETCore.App;runtime.linux-x64.Microsoft.NETCore.App;runtime.linux-musl-x64.Microsoft.NETCore.App;runtime.linux-musl-x64.Microsoft.NETCore.App;runtime.rhel.6-x64.Microsoft.NETCore.App;runtime.rhel.6-x64.Microsoft.NETCore.App;runtime.osx-x64.Microsoft.NETCore.App;runtime.osx-x64.Microsoft.NETCore.App;runtime.freebsd-x64.Microsoft.NETCore.App;runtime.freebsd-x64.Microsoft.NETCore.App;runtime.win-x86.Microsoft.NETCore.App;runtime.win-x86.Microsoft.NETCore.App;runtime.win-arm.Microsoft.NETCore.App;runtime.win-arm.Microsoft.NETCore.App;runtime.win-arm64.Microsoft.NETCore.App;runtime.win-arm64.Microsoft.NETCore.App;runtime.linux-arm.Microsoft.NETCore.App;runtime.linux-arm.Microsoft.NETCore.App;runtime.linux-arm64.Microsoft.NETCore.App;runtime.linux-arm64.Microsoft.NETCore.App;runtime.tizen.4.0.0-armel.Microsoft.NETCore.App;runtime.tizen.4.0.0-armel.Microsoft.NETCore.App;runtime.tizen.5.0.0-armel.Microsoft.NETCore.App;runtime.tizen.5.0.0-armel.Microsoft.NETCore.App;runtime.win-x64.Microsoft.NETCore.App;runtime.win-x64.Microsoft.NETCore.App";
+                    }
                 }
             }
 
+            HashSet<string> frameworkReferenceNames = new HashSet<string>(FrameworkReferences.Select(fr => fr.ItemSpec), StringComparer.OrdinalIgnoreCase);
+            RuntimeFrameworksToRemove = RuntimeFrameworks.Where(rf => !frameworkReferenceNames.Contains(rf.GetMetadata(MetadataKeys.FrameworkName)))
```

Comment:
just question, you need to remove RuntimeFrameworks?

Reply:
ResolveFrameworkReferences now resolves all the possible shared frameworks that could be referenced.  This is because eventually we will support [transitive shared framework references](https://github.com/dotnet/cli/issues/10666).  It won't be until we run ResolvePackageAssets that we know for sure which ones will be referenced, so we need to filter the targeting packs, runtime packs, and RuntimeFrameworks after that target runs.